tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -14,6 +14,8 @@ Standard output:
 ../../../../built/local/lib.es5.d.ts(1346,11): error TS2300: Duplicate identifier 'ArrayLike'.
 ../../../../built/local/lib.es5.d.ts(1382,6): error TS2300: Duplicate identifier 'Record'.
 ../../../../node_modules/@types/node/index.d.ts(150,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'module' must be of type '{}', but here has type 'NodeModule'.
+definitions.js(3,5): error TS2322: Type '{}' is not assignable to type 'typeof Animation'.
+  Property 'AnimationGroupPreviewUI' is missing in type '{}'.
```

Comment:
new, apparently we can now resolve Animation's real declaration in the DOM (maybe this is a result of a recent DOM change?). The fix is to delete `var Animation = {}` from definitions.js.

Reply:
The PR is now available at #26494